docs/fidesops/docs/guides/policies.md

```diff
@@ -107,27 +101,26 @@ PUT /api/v1/policy/{policy_key}/rule
   },
 ]
 ```
-This will create a rule to hash a not-yet-specified value with a SHA-512 hash. We need to specify a target to hash, so we need to creat a target for this rule:
 
-```json
-PUT api/v1/policy/{policy_key}/rule/{rule_key}
-  [
-    {
-      "data_category": "user.provided.identifiable.contact.email",
-    },
-  ]
+Now we add the target -- it's the same target that we used in the access Rule:
+
 ```
+PUT /api/v1/policy/{policy_key}/rule/{rule_key}/target
 
-This policy, `user_email_address_polcy`, will now do the following:
-- Return all data inside the Identity Graph with a data category that matches (or is nested under) `user.provided.identifiable.contact`.
-- Mask all data inside the Identity Graph with a data category that matches `user.provided.identifiable.contact.email` with a the `SHA-512` hashing function.
+[
+  {
+    "name": "Mask User Email Address Target",
+    "key": "mask_user_email_address_target",
+    "data_category": "user.provided.identifiable.contact.email",
+  }
+]
+```
 
-### A Note About Erasing Data
+## A Note About Masking
 
-When a Policy Rule erases data, it erases the _entire_ branch given by the Target. For example, if we created an `erasure` rule with a Target of `user.provided.identifiable.contact`, _all_ of the information within the `contact` node -- including `user.provided.identifiable.contact.email` -- would be erased.
+When a Policy Rule masks data, it masks the _entire_ branch given by the Target. For example, if we created an `erasure` rule with a Target of `user.provided.identifiable.contact`, _all_ of the information within the `contact` node -- including `user.provided.identifiable.contact.email` -- would be masked.
```

Comment:
Is it confusing to use `obfuscate` above, and `mask` here? Should we stick to one?